src/detect-app-layer-event.c

```diff
@@ -66,6 +66,8 @@ static int g_applayer_events_list_id = 0;
 void DetectAppLayerEventRegister(void)
 {
     sigmatch_table[DETECT_AL_APP_LAYER_EVENT].name = "app-layer-event";
+    sigmatch_table[DETECT_AL_APP_LAYER_EVENT].desc = "Match on events generated by the App Layer Parsers and the protocol detection engine.";
```

Comment:
nit: I'm pretty sure I had suggested the use of `Match` instead of `match` but I think that was incorrect advice as none of the existing `desc` values start with an uppercase letter. I suggest we follow convention to be consistent and thus, suggest using `match`

Reply:
Thanks, Jeff. Shall address this in the next PR.

Reply:
```git grep '\.desc ='``` shows that no trailing dot is the common pattern, so lets remove it here too.

Reply:
Ah I missed that this PR adds them. I have no objection to this and will be merging this PR. Could you do a follow up PR that changes all existing ones? There are quite few still in the old notation.